main.py

```diff
@@ -1,5 +1,31 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Genre, Actor
+
 
 def main():
-    pass
+    Genre.objects.create(name='Western')
+    Genre.objects.create(name='Action')
+    Genre.objects.create(name="Dramma")
+
+    Actor.objects.create(first_name="George", last_name="Klooney")
+    Actor.objects.create(first_name="Kianu", last_name="Reaves")
+    Actor.objects.create(first_name="Scarlett", last_name="Keegan")
+    Actor.objects.create(first_name="Will", last_name="Smith")
+    Actor.objects.create(first_name="Jaden", last_name="Smith")
+    Actor.objects.create(first_name="Scarlett", last_name="Johansson")
```

Comment:
Why you are don't scared Python's God? Better create something like set or dict or list for data and make create with for

Reply:
I just learn a new step and try to keep it simple.